README.md

```diff
@@ -74,12 +74,6 @@ required for these tests.
 
 ```
 $ pip install pipenv
```

Comment:
You don't need this line.. `pipenv run` wil create the virtual environment and install the dependencies listed in the `Pipfile.lock` for you.